commitlintplugins.test.ts

```diff
@@ -219,6 +219,23 @@ test('type-space-after-comma2', () => {
     expect(typeSpaceAfterComma2.status).toBe(0);
 });
 
+test('empty-wip-1', () => {
```

Comment:
@parhamsaremi nit: please move tests up (they go in alphabetical order)

Reply:
Done